model/version.go

```diff
@@ -741,7 +741,7 @@ func constructManifest(v *Version, projectRef *ProjectRef, moduleList ModuleList
 			// Otherwise, retrieve the module's commit from the time of the patch creation.
 			revisionTime := time.Unix(0, 0)
 			if !evergreen.IsPatchRequester(v.Requester) {
-				revisionTime = commit.Commit.Committer.GetDate()
+				revisionTime = *commit.Commit.Committer.Date.GetTime()
```

Comment:
Have we verified that this returns things in the same way? I'm also concerned about nil pointer errors so we may want to make a thirdparty helper function that returns this time and returns the empty time if it's nil or something.

Reply:
verified by making a mainline commit on sandbox without erroring 

a wrapper function shouldn't be needed with the current method because it already nil checks on both the committer and date.